core/src/main/java/apoc/trigger/Trigger.java

```diff
@@ -121,15 +126,17 @@ public Stream<TriggerInfo> pause(@Name("name")String name) {
                 (Map<String,Object>) paused.get("params"),true, true));
     }
 
-    @Procedure(name = "apoc.trigger.resume", mode = Mode.WRITE)
+    @Deprecated
+    @Procedure(name = "apoc.trigger.resume", mode = Mode.WRITE, deprecatedBy = "apoc.trigger.start")
     @Description("Resumes the given paused trigger.")
     public Stream<TriggerInfo> resume(@Name("name")String name) {
+        preprocessDeprecatedProcedures();
+        
         Map<String, Object> resume = triggerHandler.updatePaused(name, false);
 
         return Stream.of(new TriggerInfo(name,
                 (String)resume.get("statement"),
                 (Map<String,Object>) resume.get("selector"),
                 (Map<String,Object>) resume.get("params"),true, false));
     }
-
 }
```

Comment:
In total the repository seems to have 3 methods called toTriggerInfo, two constructors for TriggerInfo, and also a method from which are all instantiaters for TriggerInfo. 

I haven't properly checked but this seems like quite a lot.
- Is there anyway this could be simplified? 
- Would it make sense to move them all to the TriggerInfo class to keep the code more maintainable?